.readthedocs.yaml

```
@@ -2,7 +2,7 @@ version: 2
 build:
   os: ubuntu-20.04
```

GitHub Actions are just about to drop u20.
```suggestion
  os: ubuntu-latest
```

---

i changed it to `ubuntu-lts-latest` like in their doc: https://docs.readthedocs.com/platform/stable/config-file/v2.html#build-os